lib/matplotlib/animation.py

```diff
@@ -1202,25 +1371,75 @@ def _draw_frame(self, artists):
 
 
 class FuncAnimation(TimedAnimation):
-    '''
-    Makes an animation by repeatedly calling a function *func*, passing in
-    (optional) arguments in *fargs*.
+    '''Makes an animation by repeatedly calling a function ``func``
+
+
+    Parameters
+    ----------
+    fig : matplotlib.figure.Figure
+       The figure object that is used to get draw, resize, and any
+       other needed events.
+
+    func : callable
+
+       The function to call at each frame.  The first argument will
+       be the next value in ``frames``.   Any additional positional
+       arguments can be supplied via the ``fargs`` parameter.
+
+       The required signature is ::
+
+          def func(fr: object, *fargs) -> iterable_of_artists:
+
+
```

Comment:
Not Sure if this is an extra blank line or one needed to surround the above code block...